src/translator/service.h

```diff
@@ -161,6 +161,9 @@ class Service {
   /// Shortlist memory passed as bytes.
   AlignedMemory shortlistMemory_; // ORDER DEPENDENCY (translators_)
 
+  /// QE coefficients memory passed as bytes.
+  AlignedMemory qualityEstimator_; // ORDER DEPENDENCY (translators_)
```

Comment:
This should've been named `qualityEstimatorMemory_` for better readability (I'm constantly confusing this name with a `QualityEstimator` instance).

But the change here is perhaps to replace `AlignedMemory` here with an instance of `QualityEstimator`.

Reply:
done:  064cfe3